includes/class-algolia-cli.php

```diff
@@ -120,7 +125,7 @@ private function do_reindex( Algolia_Index $index, $clear ) {
 			WP_CLI::success( sprintf( __( 'Correctly cleared index "%s".', 'wp-search-with-algolia' ), $index->get_name() ) );
 		}
 
-		$total_pages = $index->get_re_index_max_num_pages();
+		$total_pages = $index->get_re_index_max_num_pages() - ( $from_page - 1 );
```

Comment:
While I'm sure the reason to do the math is a legit one, can you explain the need for the subtracting of 1 from the page number set in the variable?

Reply:
Let's assume we have 10 pages in total so that `$index->get_re_index_max_num_pages()` returns 10.

- If you start from page 8 (`$from_page = 8`), you will run 3 pages (8, 9 and 10). `10 - (8 - 1) == 3`
- If you start from page 1 (`$from_page = 1`), you will run 10 pages. `10 - (1 - 1) == 10`
- 
Does that make it clear? I really tried (even before your comment) to write a comment in the code for this since it does look counter intuitive. My english is not good enough to explain it clearly though. If you have a suggestion for how to write I'll happily add the comment.